<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SGFPreview</class>
 <widget class="QWidget" name="SGFPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <layout class="QGridLayout" name="gridLayout_2"/>
   </item>
   <item row="0" column="0" rowspan="4">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,0">
     <item>
      <widget class="QWidget" name="dialogWidget" native="true"/>
     </item>
     <item>
      <widget class="QWidget" name="archiveItemListContainer" native="true"/>
     </item>
     <item>
      <widget class="QGroupBox" name="overwriteSGFEncoding">
       <property name="title">
        <string>Override character set for text in SGF file:</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="encodingList">
          <item>
           <property name="text">
            <string>Auto Detect</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>UTF-8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GB18030</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EUC-KR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EUC-JP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ISO-2022-JP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Shift-JIS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Big5</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="FigureView" name="boardView">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="6" column="2">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Komi:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="q4go.qrc">:/images/stone_b16.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="5">
      <widget class="QLabel" name="File_BlackPlayer">
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Black</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Handicap:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="5">
      <widget class="QLabel" name="File_Date">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Result:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="5">
      <widget class="QLabel" name="File_Handicap">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="File_Size">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="text">
        <string>19</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="5">
      <widget class="QLabel" name="File_WhitePlayer">
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="text">
        <string>White</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="q4go.qrc">:/images/stone_w16.png</pixmap>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Date:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="File_Komi">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="text">
        <string>6.5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="5">
      <widget class="QLabel" name="File_Result">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Event:</string>
       </property>
       <property name="buddy">
        <cstring>File_Event</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QLabel" name="File_Event">
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Round:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QLabel" name="File_Round">
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FigureView</class>
   <extends>QGraphicsView</extends>
   <header>board.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="q4go.qrc"/>
 </resources>
 <connections/>
</ui>
